routes/tasks.js

```diff
@@ -155,4 +155,51 @@ router.patch('/:id', authenticate, authorizeUser('appOwner'), updateTask, tasks.
  */
 router.get('/:username', tasks.getUserTasks)
 
+/**
+ * @swagger
+ * /tasks/self/:id:
+ *  patch:
+ *   summary: used to update self task status
+ *   tags:
+ *     - Tasks
+ *   requestBody:
```

Comment:
are you sure this is `requestBody`?

Reply:
I checked in other routes, for post or patch it's requestBody. Am I missing some points here?

Reply:
Cool then